config.go

```diff
@@ -233,6 +237,7 @@ func SetupDaemonConfig(logger *logrus.Logger, configFile string) (DaemonConfig,
 	// Main config
 	setter.SetDefault(&conf.GRPCListenAddress, os.Getenv("GUBER_GRPC_ADDRESS"), "localhost:81")
 	setter.SetDefault(&conf.HTTPListenAddress, os.Getenv("GUBER_HTTP_ADDRESS"), "localhost:80")
+	setter.SetDefault(&conf.GRPCMaxConnectionAgeSeconds, os.Getenv("GUBER_GRPC_MAX_CONN_AGE_SEC"), 0)
```

Comment:
This should use `getEnvInteger(log, "GUBER_GRPC_MAX_CONN_AGE_SEC")`, Additionally I think `GRPCMaxConnectionAgeSeconds` should be an `int` instead of an `int64`.

Reply:
Hello, thanks for this review. 

All suggestions have been applied.